src/iris/vendors/iris_slack.py

```diff
@@ -125,8 +126,11 @@ def send_message(self, message):
                     return time.time() - start
                 # If message is invalid:
                 #   {u'ok': False, u'error': u'invalid_arg_name'}
-                logger.error('Received an error from slack api: %s',
-                             data['error'])
+                # if not in the channel, the error is expected so log a warning instead of an error
+                elif data.get('error') == 'not_in_channel':
+                    logger.warning('Iris bot not present in the designated channel %s', message.get('destination'))
```

Comment:
I believe you have to return a runtime `return time.time() - start` or else it will still be counted in the failure metrics